modules/xfem/src/userobjects/MeshCut2DFractureUserObject.C

```diff
@@ -149,12 +149,34 @@ MeshCut2DFractureUserObject::findActiveBoundaryGrowth()
       Real k_squared = _ki_vpp->at(i) * _ki_vpp->at(i) + _kii_vpp->at(i) * _kii_vpp->at(i);
       if (k_squared > (k_crit * k_crit) && _ki_vpp->at(i) > 0)
       {
-        // growth direction in crack front coord (cfc) system based on the  max hoop stress
-        // criterion
+        // growth direction in crack front coord (cfc) system based on the max hoop stress criterion
+        // Ref "Ceramic nuclear fuel fracture modeling with the extended finite element method" by
+        // W. Jiang, B. Spencer, and J. Dolbow
+        // Equation 6
         Real ki = _ki_vpp->at(i);
         Real kii = _kii_vpp->at(i);
-        Real sqrt_k = std::sqrt(ki * ki + kii * kii);
-        Real theta = 2 * std::atan((ki - sqrt_k) / (4 * kii));
+        Real sqrt_k = std::sqrt(ki * ki + 8 * kii * kii);
+        Real theta_m = 2 * std::atan((ki - sqrt_k) / (4 * kii)); // a more stable implementation
+        Real theta_p = 2 * std::atan((ki + sqrt_k) / (4 * kii)); // a more stable implementation
+
+        // check sigma_tt
+        // Ref "Ceramic nuclear fuel fracture modeling with the extended finite element method" by
+        // W. Jiang, B. Spencer, and J. Dolbow
+        // Equation 5
+        Real _r = _growth_increment;
```

Comment:
```suggestion
        Real r = _growth_increment;
```

Reply:
The "_" is used to identify something as a member variable.  So remove those from temporary variables.